external.bzl

```diff
@@ -31,7 +31,7 @@ def _rule_dependencies():
     rules_proto_dependencies()
     py_repositories()
     bazel_toolchains_repositories()
-    rust_repositories(version = "1.43.1")
+    rust_repositories(version = "nightly", iso_date = "2020-06-23", dev_components = True)
```

Comment:
Hm you normally don't want to depend on tags that can be moved.  *If* you can do with a hash instead, it would probably be better.

Reply:
Unfortunately `iso_date` is the most specific field that can be set, and we require the nightly builds to get the rustc_dev component. However, the hosting for nightly builds is stable because a lot of the community depends on them for their builds. For example, the packages for this nightly build are hosted at https://static.rust-lang.org/dist/2020-06-23